monai/networks/nets/torchvision_fc.py

```diff
@@ -9,66 +9,56 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from typing import Tuple, Union
-
-import torch
+from typing import Any, Dict, Optional, Tuple
 
+from monai.networks.nets import NetAdapter
 from monai.utils import optional_import
 
 models, _ = optional_import("torchvision.models")
 
 
-__all__ = ["TorchVisionFullyConvModel"]
+__all__ = ["TorchVisionFCModel"]
 
 
-class TorchVisionFullyConvModel(torch.nn.Module):
+class TorchVisionFCModel(NetAdapter):
```

Comment:
This might break an existing pathology MMAR, need to create an alias for the old name?

Reply:
I feel no need to create a alias, all the MMAR should totally update when adapting to a new Clara version later.
What do you think?

Thanks.

Reply:
but we want to support mmar downloading in the next monai release... merging this PR will break the dev test at the moment https://github.com/Project-MONAI/MONAI/blob/c20f70bf8c36c09630f7add95f823dbbcf1e9a73/tests/test_mmar_download.py#L87
maybe it's a good opportunity to think about backward comp. now

Reply:
OK, let me create an alias class for it.
Thanks for letting me know the potential issue.

Reply:
Added back `TorchVisionFullyConvModel` in the latest commit for compatibility.
Will remove it in v0.7.0 when the MMAR updated to use new `TorchVisionFCModel` API.

Thanks.